### PR TITLE
dev-util/ebuildtester: add 44 

### PR DESCRIPTION
**Pull request by stkw0**
Also add myself as maintainer, if @nicolasbock is ok with that.

---

Please check all the boxes that apply:

- [x] I can submit this contribution in agreement with the [Copyright Policy](https://www.gentoo.org/glep/glep-0076.html#certificate-of-origin).
- [x] I have certified the above via adding a `Signed-off-by` line to *every* commit in the pull request.
- [x] This contribution has not been created with the assistance of Natural Language Processing artificial intelligence tools, in accordance with the [AI policy](https://wiki.gentoo.org/wiki/Project:Council/AI_policy).
- [x] I have run `pkgcheck scan --commits --net` to check for issues with my commits.

Please note that all boxes must be checked for the pull request to be merged.

### CODE REVIEW

File: dev-util/ebuildtester/Manifest [hunk 1]
```diff
@@ -1 +1,2 @@
 DIST ebuildtester-42.tar.gz 22852 BLAKE2B 0e672c70ed854b3ee459010590da22811ed06e0397d45e51b8ad1a86047aa3fbd61321a7a9bdf1b583c3a66246030bb30dbef4dc8acc1e10bc3b141095e08ff4 SHA512 c2fe0dbca673c51d686cc19954928efb3ab1e82c4d8b8548d272f8fc60a5fffd1898fe33dc26feebc5bc73a71207f5f1615e18036b264e76a09b9dc6d94168fa
+DIST ebuildtester-44.tar.gz 23982 BLAKE2B 96ebdaa1278068d83805b3d9b86b0191ded2028dc042c3ed5dbd22cee38831f07f8a69a3e9ba955d189b64b20b17669f2587ab259cf4f80b490a055bbbf98053 SHA512 a391821999fe3b559dddb6bda326e25d8b64e59330b87fe7f0453aefcf4145720d72bde562d791c147b7a9c3901a676a7184db3b8ce1c8b5f8923ca3473b09ef
```
Comment (eli-schwartz) [inline dev-util/ebuildtester/Manifest:2]: You're removing the old version but not removing the old version from Manifest?

File: dev-util/ebuildtester/metadata.xml [hunk 2]
```diff
@@ -4,6 +4,14 @@
 	<maintainer type="person">
 		<email>nicolasbock@gentoo.org</email>
 	</maintainer>
+	<maintainer type="person" proxied="yes">
+		<email>davidroman96@gmail.com</email>
+		<name>David Roman</name>
```
Comment (eli-schwartz) [inline dev-util/ebuildtester/metadata.xml:9]: I would also suggest doing this in a third commit, not the "add" one. Among other things, this makes it easier to trivially revert, or to cherry-pick.

**Review by eli-schwartz — CHANGES_REQUESTED** [review]

File: dev-util/ebuildtester/Manifest (hunk 1 above)
Comment (stkw0) [reply dev-util/ebuildtester/Manifest:2]: As there was two revisions for the same version, even when I removed one revision the other revision still downloads the same tar file and hence it is still present in the manifest file.

File: dev-util/ebuildtester/metadata.xml (hunk 2 above)
Comment (stkw0) [reply dev-util/ebuildtester/metadata.xml:9]: done :)

**Review by Jamim — CHANGES_REQUESTED** [review]
_Hello @stkw0,_

Would you mind adding [the repository url](https://github.com/nicolasbock/ebuildtester) to the `HOMEPAGE` variable?
```diff
-HOMEPAGE="https://ebuildtester.readthedocs.io/"
+HOMEPAGE="https://ebuildtester.readthedocs.io https://github.com/nicolasbock/ebuildtester"
```

_Thanks!_

File: dev-util/ebuildtester/Manifest (hunk 1 above)
Comment (eli-schwartz) [reply dev-util/ebuildtester/Manifest:2]: Yes, sorry for being dumb. :o

**Comment by eli-schwartz** [comment]
@Jamim any particular reason, given it's already set in metadata.xml as a remote id?

**Comment by Jamim** [comment]
> @Jamim any particular reason, given it's already set in metadata.xml as a remote id?

It's just more convenient since [eix](https://github.com/vaeth/eix) allows you to quickly show the `HOMEPAGE`.
```shell
$ eix ebuildtester
* dev-util/ebuildtester
     Available versions:  (~)42-r2^t (~)42-r3^t {doc test PYTHON_TARGETS="python3_10 python3_11 python3_12"}
     Homepage:            https://ebuildtester.readthedocs.io/
     Description:         A dockerized approach to test a Gentoo package within a clean stage3 container
```

**Comment by Flowdalic** [comment]
> It's just more convenient since [eix](https://github.com/vaeth/eix) allows you to quickly show the `HOMEPAGE`.

I actually just wondered about the same, but with a different package. I think I would personally prefer to improve `eix` so that it also displays the source URL acquired from `metadata.xml` instead of adding an additional homepage.

Of course, this is just my personal opinion and, as far as I can tell, there is no Gentoo policy on that on how to deal with this case. So you may do what you think is best.

**Comment by thesamesam** [comment]
> I actually just wondered about the same, but with a different package. I think I would personally prefer to improve eix so that it also displays the source URL acquired from metadata.xml instead of adding an additional homepage.

See https://github.com/vaeth/eix/issues/105.

> It's just more convenient since [eix](https://github.com/vaeth/eix) allows you to quickly show the HOMEPAGE.

I'd prefer to not overdo it and always duplicate it. If it's especially helpful, I guess it's OK, but I wouldn't want people to default to doing it. 

iwdevtools' [repo-cd](https://github.com/ionenwks/iwdevtools?tab=readme-ov-file#repo-cd) works well here for when you're working on an ebuild and want to quickly get to VCS.

> Also add myself as maintainer, if @nicolasbock is ok with that.

@Flowdalic, where did @nicolasbock ack it?

**Comment by nicolasbock** [comment]
@thesamesam the change looks good. Thanks for pinging me.

**Comment by Flowdalic** [comment]
> See [vaeth/eix#105](https://github.com/vaeth/eix/issues/105).

Oh, what a pity. :/

> > Also add myself as maintainer, if @nicolasbock is ok with that.
> 
> @Flowdalic, where did @nicolasbock ack it?

I somehow assumed that this was acked. This was obviously not the case and I should not have merged it without the ack. Sorry for that.

> @thesamesam the change looks good. Thanks for pinging me.

@nicolasbock Thanks for ebuildtester, I use that tool once in a while and it usually comes in handy.

**Comment by nicolasbock** [comment]
@Flowdalic I'm glad to hear that you find the tool useful 🙂

Thanks!

**Pull request closed without merging**